backend/scripts/cleanup_runtime_db.sh

```
@@ -70,8 +73,8 @@ DB_PATH=""
 USERNAME=""
 PASSWORD=""
 
-# Tables to clean (order matters: FLOW_CONTEXT first for cascade).
-TABLES=("FLOW_CONTEXT" "AUTHORIZATION_CODE" "AUTHORIZATION_REQUEST" "WEBAUTHN_SESSION" "ATTRIBUTE_CACHE" "PAR_REQUEST" "JTI_RECORD" "OPENID4VP_REQUEST_STATE" "OPENID4VCI_NONCE" "OPENID4VCI_CREDENTIAL_OFFER")
+# Tables to clean
+TABLES=("AUTHORIZATION_CODE" "AUTHORIZATION_REQUEST" "WEBAUTHN_SESSION" "ATTRIBUTE_CACHE" "PAR_REQUEST" "JTI_RECORD" "OPENID4VP_REQUEST_STATE" "OPENID4VCI_NONCE" "OPENID4VCI_CREDENTIAL_OFFER" "RUNTIME_STORE"")
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this array assignment.

(SC1009)